Description: link with external abseil
 It's not embedded in source.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Last-Update: 2024-08-11

---

Index: grpc/setup.py
===================================================================
--- grpc.orig/setup.py
+++ grpc/setup.py
@@ -169,7 +169,7 @@ BUILD_WITH_SYSTEM_RE2 = _env_bool_value(
 # Export this variable to use the system installation of abseil. You need to
 # have the header files installed (in /usr/include/absl) and during
 # runtime, the shared library must be installed
-BUILD_WITH_SYSTEM_ABSL = os.environ.get("GRPC_PYTHON_BUILD_SYSTEM_ABSL", False)
+BUILD_WITH_SYSTEM_ABSL = os.environ.get("GRPC_PYTHON_BUILD_SYSTEM_ABSL", True)
 
 # Export this variable to force building the python extension with a statically linked libstdc++.
 # At least on linux, this is normally not needed as we can build manylinux-compatible wheels on linux just fine
